cypress/integration/endpoint-tests.js

```diff
@@ -8,7 +8,8 @@ describe(`Visual regression testing for foundation.mozilla.org`, () => {
   // Main pages
 
   it(`Foundation homepage`, function() {
-    cy.visit(`/`);
+    cy.visit(`/en/`);
```

Comment:
seriously? that's what did it?

Reply:
nope, but it was redirecting anyways. The fix was to update the cypress tests to use `localhost` instead of `0.0.0.0`